### PR TITLE
IIS experience

### PR DESCRIPTION
**Pull request by p3ck**
This experience is for deploying Windows IIS Server.
    
Include the execution environment to run against and how to build it manually for now.

### CODE REVIEW

**Comment by p3ck** [comment]
Please give this another look.  I incorporated most of your suggestions but I still had questions on some of them. (if you look at the original pr here : https://github.com/p3ck/infra.windows_ops.orig/pull/1

Thanks

File: extensions/experiences/create_iis/exec_env/requirements.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+collections:
+  # - name: ansible.experience_demo
+  - name: https://github.com/p3ck/infra.windows_ops.git
```
Comment (mhjacks) [inline extensions/experiences/create_iis/exec_env/requirements.yml:4]: We can update this now, thankfully :)

File: extensions/experiences/create_iis/playbooks/run_create_iis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+- name: Create IIS Server Play
+  hosts: all
+  gather_facts: false
+  tasks:
+    - name: Create default test message if not provided
+      ansible.builtin.set_fact:
+        test_message: "Welcome to IIS Web Server"
```
Comment (mhjacks) [inline extensions/experiences/create_iis/playbooks/run_create_iis.yml:8]: I frankly miss the Rick Astley quote, but this is probably better

File: extensions/experiences/create_iis/setup.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    organization: "{{ organization | default('Default') }}"
+
+# Execution Environments
+#
+controller_execution_environments:
+  # - name: apd-ee-25
+  #   description: apd-ee-25
+  #   image: quay.io/ansible-product-demos/apd-ee-25
+  #   pull: always
+  - name: apd-ee-25-rhel
```
Comment (mhjacks) [inline extensions/experiences/create_iis/setup.yml:19]: Why comment the "main" APD image here? The -rhel tag seems off in this context.

File: extensions/experiences/delete_iis/exec_env/requirements.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+collections:
+  # - name: ansible.experience_demo
+  - name: https://github.com/p3ck/infra.windows_ops.git
```
Comment (mhjacks) [inline extensions/experiences/delete_iis/exec_env/requirements.yml:4]: Should be able to update this pointer now too

**Review by mhjacks — CHANGES_REQUESTED** [review]
I think we have a couple of references to the interim repo that have to change. I do not see anything else I'd consider blocking, and additionally I think there is value in getting  into the test workflow  ASAP as I expect we will learn other things about experience semantics in doing so (and we still have time to finalize content).

File: extensions/experiences/create_iis/exec_env/requirements.yml (hunk 1 above)
Comment (p3ck) [reply extensions/experiences/create_iis/exec_env/requirements.yml:4]: I will update it now and point it to main for the branch.  But until it's merged it only existed in my repo's branch. :-)

File: extensions/experiences/create_iis/setup.yml (hunk 3 above)
Comment (p3ck) [reply extensions/experiences/create_iis/setup.yml:19]: Good point.  You can see I copy and pasted this from the rhel experience.

**Pull request merged**